docs/integration/resources.md

```diff
@@ -0,0 +1,240 @@
+typing these identifiers in manually--they'll typically obtain them by searching
+your database and selecting desired results.
+
+The DTS assigns a database specific prefix to your identifier that gives it a
+_namespace_ inside which it is unique. For example, the JGI Data Portal
+identifier `615a383dcc4ff44f36ca5ba2` is made available by the DTS as
+`JDP:615a383dcc4ff44f36ca5ba2`.
+
+Sometimes a database consists of more than one dataset, and each dataset has its
+own identifiers. For example, [Uniprot](https://www.uniprot.org/), one of
+the world's largest collections of protein sequences, defines unique identifiers
+for each of several distinct datasets. [This link](https://www.uniprot.org/help/linking_to_uniprot)
+illustrates some examples of these dataset-specific identifiers.
+
+Because Uniprot's identifiers indicate which dataset they're from, all of the
+identifiers could be used within a single namespace by the DTS. This is one way
+of providing access to multiple datasets. If this method of aggregating datasets
+into a single namespace doesn't work for your organization, the DTS team is
+happy to discuss alternatives with you.
+
+## File Metadata
+
+Whatever file identification scheme your organization uses, you can provide the
+DTS with the file information it needs by creating an endpoint that returns
+file metadata for a given unique file identifier.
+
+What's "metadata"? There is no universal answer to this question, of course. The
+DTS is part of an effort to harmonize how our community searches for, obtains,
+and cites scientific data, but the size and diversity of this community make it
+difficult to find an answer even for our purposes alone!
+
+Nevertheless, we press on. The metadata specification used by the DTS is likely
+to undergo changes of all sorts as we figure things out, but we'll give our best
+effort to updating documentation as these changes occur.
+
+### DTS Metadata Specification
+
+The DTS stores file metadata in the [Frictionless DataResource](https://specs.frictionlessdata.io/data-resource/)
+format. This format is simple and relatively unopinionated, and allows for
+additional fields as needed. The most important elements of the Frictionless
+DataResource type are:
+
+* `name`: the name of a resource (file)
+* `path`: the path of the resource relative to the root directory of the
+  [staging area](staging_area.md) in which you make the file available
+
+The following fields are used by the DTS in addition to the ones described in
+the link:
+
+* `id`: your organization's unique identifier for the resource
+* `credit`: credit metadata associated with the resource that conforms to the
+  [KBase credit metadata schema](https://github.com/kbase/credit_engine)
+* `metadata`: an optional unѕtructured field that you can use to stash
+  additional information about the resource if needed. Don't worry if you can't
+  think of a use for this one.
```

Comment:
not sure that the second sentence is helpful. May be better to say something about expecting usage of this field to develop as more use cases arise